tests/conftest.py

```diff
@@ -32,3 +34,35 @@ def sonata_config():
             "tstop": 10,
         }
     )
+
+
+@pytest.fixture(autouse=True)
+def change_test_dir(monkeypatch, tmp_path):
+    monkeypatch.chdir(tmp_path)
+
+
+@pytest.fixture
+def create_tmp_simulation_file(request, tmp_path):
+    """ copy simulation config file to tmp_path
+    """
+    params = request.param
+    src_dir = Path(params.get("src_dir"))
+    extra_config = params.get("extra_config")
+    config_file = Path(params.get("simconfig_file"))
+    with open(str(src_dir / config_file)) as src_f:
```

Comment:
I wonder if it's worthwhile to have a `load_json` helper, since this pattern gets used a lot in the codebase

Reply:
I am still thinking about it. A helper that replaces these lines ? Is it really needed? 
```
    with open(str(src_dir / config_file)) as src_f:
        sim_config_data = json.load(src_f)
```

Reply:
No worries, for now, I'll do it in another PR.